tools/sidekick/app.js

```diff
@@ -371,7 +371,7 @@
         this.config.plugins.forEach((plugin) => this.add(plugin));
       }
       if ((this.isHelix() || this.isEditor()) && this.config.innerHost) {
-        const prefix = this.isEditor() ? `https://${this.config.innerHost}` : '';
+        const prefix = this.config.pluginHost || this.isEditor() ? `https://${this.config.innerHost}` : '';
```

Comment:
No test for this case ? ;)

Reply:
You're absolutely right!

Reply:
Turns out it wasn't even working as intended 😊